src/grow_worker/worker.py

```diff
@@ -0,0 +1,102 @@
+import jsonpickle
+import os
+import logging
+from typing import Optional
+from uuid import uuid4
+from pathlib import Path
+from celery import Celery
+from dataclasses import dataclass
+import subprocess
+
+from grow_runner.types import WorkFlowType
+
+LOGGER = logging.getLogger("grow_worker")
+
+app = Celery(
+    "omotes",
+    broker="amqp://user:bitnami@rabbitmq",
+    backend="rpc://user:bitnami@rabbitmq",
+    broker_connection_retry_on_startup=True,
```

Comment:
Couple of settings still need to be set:

https://docs.celeryq.dev/en/stable/userguide/configuration.html#worker-prefetch-multiplier  should be 1
https://docs.celeryq.dev/en/stable/userguide/configuration.html#task-acks-late should be True
https://docs.celeryq.dev/en/stable/userguide/configuration.html#task-reject-on-worker-lost should be True